source/MaterialXFormat/File.cpp

```diff
@@ -122,10 +122,17 @@ FilePath FilePath::operator/(const FilePath& rhs) const
         throw Exception("Appended path must be relative.");
     }
 
+    const string DOT_PATH = ".";
```

Comment:
Can we now remove this special handling of dot paths in FilePath concatenation, since the client can normalize their paths on demand?

Reply:
Yes. Removed now.